comp/logs/agent/config/config_keys.go

```diff
@@ -95,6 +95,16 @@ func (l *LogsConfigKeys) logsNoSSL() bool {
 	return l.getConfig().GetBool(l.getConfigKey("logs_no_ssl"))
 }
 
+func (l *LogsConfigKeys) logLineMaxSize() int {
+	key := l.getConfigKey("log_line_max_size")
+	logLineMaxSize := l.getConfig().GetInt(key)
+	// max content length limit is 1MB
+	if logLineMaxSize < 1 || 1000 < logLineMaxSize {
+		log.Warnf("Invalid %s: %v should be in [1, 1000] kilobytes, fallback on %v bytes", key, logLineMaxSize, coreConfig.DefaultLogLineMaxSize)
```

Comment:
I think we can remove this warning, and the bounds check. 
This is an advanced setting that most users won't touch so I don't think we need to protect users from misconfiguring it. I would instead add a message in the config template like:

> if maxMessageSize exceeds the documented API limit of 1MB - any payloads larger than 1MB will be dropped by the intake.

And leave a link to our API docs https://docs.datadoghq.com/api/latest/logs/